GSA_Adapter/Convert/FromGsa/Loads/NodeLoad.cs

```diff
@@ -0,0 +1,146 @@
+        /***************************************************/
+
+        public static PointLoad FromGsaNodeLoad(string gsaString, Dictionary<string, Loadcase> lCases, Dictionary<string, Node> nodes, double unitFactor)
+        {
+            if (string.IsNullOrWhiteSpace(gsaString))
+                return null;
+
+            string[] gStr = gsaString.Split(',');
+
+            if (gStr.Length < 7)
+                return null;
+
+            string lCaseNo = gStr[3];
+
+            Loadcase loadCase;
+            if (!lCases.TryGetValue(lCaseNo, out loadCase))
+            {
+                loadCase = new Loadcase { Number = int.Parse(lCaseNo), Nature = LoadNature.Other };
+                loadCase.SetAdapterId(typeof(GSAId), int.Parse(lCaseNo));
+            }
+
+            string[] nodeNos = gStr[2].Split(' ');
+            string nodeNosClean = nodeNos[0];
+            for (int i = 1; i < nodeNos.Length; i++)
+            {
+                string addNo = nodeNos[i];
+                if(nodeNos[i] == "to")
+                {
+                    addNo = "";
+                    int range = int.Parse(nodeNos[i + 1]) - int.Parse(nodeNos[i - 1]);
+                    for (int j = 1; j < range; j++)
+                    {
+                        int number = int.Parse(nodeNos[i - 1]) + j;
+                        addNo = addNo + " " + number.ToString();
+                    }
+                }
+
+                nodeNosClean = nodeNosClean + " " + addNo;
+            }
+
+            string[] nodeNosCleanArray = nodeNosClean.Split(' ');
+
+            BHoMGroup<Node> nodeGroup = new BHoMGroup<Node>();
+
+            for (int i = 0; i < nodeNosCleanArray.Length; i++)
+            {
+                string cleanStr = nodeNosCleanArray[i].Replace(" ", "");
+                if (!string.IsNullOrEmpty(cleanStr))
+                {
+                    Node node;
+                    if (!nodes.TryGetValue(cleanStr, out node))
+                    {
+                        node = new Node();
+                        node.SetAdapterId(typeof(GSAId), int.Parse(cleanStr));
+                    }
+                    nodeGroup.Elements.Add(node);
+                }
+            }
+
+            double fx = 0;
+            double fy = 0;
+            double fz = 0;
+            double mx = 0;
+            double my = 0;
+            double mz = 0;
+     
+            if (gStr[5] == "X")
+                fx = double.Parse(gStr[6]) / unitFactor;
+            if (gStr[5] == "Y")
+                fy = double.Parse(gStr[6]) / unitFactor;
+            if (gStr[5] == "Z")
+                fz = double.Parse(gStr[6]) / unitFactor;
+            if (gStr[5] == "XX")
+                mx = double.Parse(gStr[6]) / unitFactor;
+            if (gStr[5] == "YY")
+                my = double.Parse(gStr[6]) / unitFactor;
+            if (gStr[5] == "ZZ")
+                mz = double.Parse(gStr[6]) / unitFactor;
```

Comment:
```suggestion
            double convertedFactor = double.Parse(gStr[6]) / unitFactor;
            
            if (gStr[5] == "X")
                fx = convertedFactor;
            else if (gStr[5] == "Y")
                fy = convertedFactor;
            else if (gStr[5] == "Z")
                fz = convertedFactor;
            else if (gStr[5] == "XX")
                mx = convertedFactor;
            else if (gStr[5] == "YY")
                my = convertedFactor;
            else if (gStr[5] == "ZZ")
                mz = convertedFactor;
```

This suggestion is to aid code readability by moving the calculation `double.Parse(gStr[6]) / unitFactor` to the top of the statements so it's only used once (also means if it needs to be changed later, change is only needed in one place).

Have also suggested using `else if` on the check of which property gets the converted value - only because it's always comparing `gStr[5]` and if the first `if` statement passes, it would still check the others (in the current implementation). This change means if it passes on the first, the rest aren't evaluated (giving a small performance benefit).